modules/simple_forms_api/spec/services/notification_email_spec.rb

```diff
@@ -151,7 +151,7 @@
                 email,
                 "form21_10210_#{notification_type}_email_template_id",
                 {
-                  'first_name' => 'JOHN',
+                  'first_name' => 'John',
```

Comment:
Can you change at least one of these first names in a spec to be a double first name? `'John Luke'`?

Reply:
Yep! I added one elsewhere. Thanks!